Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN apk add postgresql-client
 # add flyway
 RUN apk add --no-cache --update openjdk8-jre-base bash gettext libc6-compat
 
-ARG FLYWAY_VERSION=9.19.3
```

Comment:
Why the downgrade of flyway?

Reply:
It was a mistake, good catch reverting the line to the previous version:.